tests/specs/session-replay/preload.e2e.js

```diff
@@ -28,4 +28,47 @@ describe.withBrowsersMatching(notIE)('Session Replay Payload Validation', () =>
     testExpectedReplay({ data: harvestContents })
     expect(wasPreloaded).toEqual(true)
   })
+
+  it('should allow for preload if configured', async () => {
```

Comment:
Is this test only verifying that rrweb was loaded before page load event? Should we be testing that rrweb correctly captures data before the page load event? Or am I misunderstanding the purpose of preloading rrweb?

Reply:
That is what I meant. I don't see anything in that function that specifically is checking that the recording happened pre page load event.